spec/tictactoe_clojure/game_spec.clj

```diff
@@ -0,0 +1,46 @@
+(ns tictactoe-clojure.game-spec
+  (:require [speclj.core :refer :all]
+            [tictactoe-clojure.board :as board]
+            [tictactoe-clojure.human-player :as human-player]
+            [tictactoe-clojure.computer-player :as computer-player]
+            [tictactoe-clojure.rules :as rules]
+            [tictactoe-clojure.console :as console]
+            [tictactoe-clojure.game :refer :all]))
+
+(describe "tictactoe-clojure.game"
+  (with new-board [1 2 3 4 5 6 7 8 9])
+  (with x-win-1st-row-board ["X" "X" "X" "O" "O" 6 7 8 9])
+  (with board-with-draw ["X" "O" "X" "O" "O" "X" "X" "X" "O"])
+  (with board-displayed "1 | 2 | 3\n---------\n4 | 5 | 6\n---------\n7 | 8 | 9\n")
+
+  (describe "#take-two-moves"
+    (it "places O on the board where specified if it is Os turn"
+      (with-redefs [human-player/human-move (fn [_] 1)
+                    computer-player/optimal-move nil]
+        (should= ["O" 2 3 4 5 6 7 8 9] (take-two-moves @new-board "O")))))
+
+  (describe "#show-result"
+    (it "invokes #draw? to determine if there is a draw"
+      (with-redefs [console/display (fn [_] nil) ]
+        (should-invoke rules/draw? {:times 1} (show-result @new-board))))
+    (it "displays It's a draw if the result is a draw"
+      (with-redefs [console/display (fn [_] "draw")]
+        (should= "draw" (show-result @board-with-draw))))
+    (it "if it is not a draw #winner is invoked to check who the winner is"
+      (with-redefs [console/display (fn [_] nil) ]
+        (should-invoke rules/winner {:times 1} (show-result @new-board))))
+    (it "it there is a winner it displays the winner"
+      (with-redefs [console/display (fn [_] "X wins")]
+        (should= "X wins" (show-result @x-win-1st-row-board)))))
+
+  (describe "#play-game"
+    (it "#show-result is called when the game is over"
```

Comment:
Unfortunately, I am testing implementation here. To test for behavior, I should be passing a board where x wins and assert that the behavior should be that it outputs a message where X wins. On the side, I might have to simulate an actual game. Although I am not sure if that is the best way to test here.